### PR TITLE
Bump build/publish images and golang version

### PR DESCRIPTION
**Pull request by brandond**
Signed-off-by: Brad Davidson <brad.davidson@rancher.com>

### CODE REVIEW

File: .golangci.json [hunk 1]
```diff
@@ -3,11 +3,28 @@
 		"disable-all": true,
 		"enable": [
 			"govet",
-			"golint",
-			"goimports"
+			"revive",
+			"goimports",
+      "gofmt"
```
Comment (briandowns) [inline .golangci.json:8]: How ironic.

File: .golangci.json [hunk 2]
```diff
@@ -3,11 +3,28 @@
...
+			{
+				"linters": "typecheck",
+				"text": "imported but not used"
+			},
+			{
+				"linters": "revive",
+				"text": "should have comment"
+			},
+			{
+				"linters": "revive",
```
Comment (briandowns) [inline .golangci.json:25]: Why are we excluding this check?

File: .golangci.json (hunk 1 above)
Comment (brandond) [reply .golangci.json:8]: yay for per-file indentation conventions that my editor hides...

File: .golangci.json (hunk 2 above)
Comment (brandond) [reply .golangci.json:25]: I copied the exclude-rules from the k3s config. Let's see what happens if I don't exclude it.

**Review by briandowns — APPROVED** [review]

**Pull request merged**